packages/airnode-node/src/index.ts

```diff
@@ -1,7 +1,9 @@
-export * as handlers from './handlers';
+export * as adapters from './adapters';
 export * as config from './config';
-export * as providerState from './providers/state';
+export * as evm from './evm';
+export * as handlers from './handlers';
 export * as logger from './logger';
-export * as promiseUtils from './utils/promise-utils';
```

Comment:
Changing these imports is going to break stuff. I'm pretty sure things like the deployer depends on these so you'll need to check that out

Reply:
Build and tests are passing. I also tested the airnode-operation commands and they are working as well. I'll try running the deployer cli.

Reply:
@andreogle is right, and the build is not passing. Maybe you looked at some outdated build?